invokeai/backend/model_manager/load/load_default.py

```diff
@@ -123,15 +104,39 @@ def get_size_fs(
             variant=config.repo_variant if isinstance(config, DiffusersConfigBase) else None,
         )
 
+    def _do_convert(
+        self, config: AnyModelConfig, model_path: Path, cache_path: Path, submodel_type: Optional[SubModelType] = None
+    ) -> AnyModel:
+        if self._convert_cache.max_size <= 0:  # no caching
+            pipeline = self._convert_model(config, model_path)
+            if submodel_type:
+                for subtype in SubModelType:
+                    if subtype == submodel_type:
+                        continue
+                    # cache other parts of the model so that we don't re-convert
+                    if submodel := getattr(pipeline, subtype.value, None):
+                        self._ram_cache.put(
+                            config.key, submodel_type=subtype, model=submodel, size=calc_model_size_by_data(submodel)
+                        )
+            loaded_model = getattr(pipeline, submodel_type.value) if submodel_type else pipeline
+        else:
+            self._convert_model(config, model_path, cache_path)
+            config.path = cache_path.as_posix()
+            loaded_model = self._load_model(config, submodel_type)
+
+        return loaded_model
+
+    def _needs_conversion(self, config: AnyModelConfig, model_path: Path, dest_path: Path) -> bool:
+        return False
```

Comment:
What is this doing?

Reply:
Setting a default for the `_needs_conversion()` method. Most models don't support conversion. The few that do override this method and have logic to determine whether the conversion needs to be performed (e.g. looks in the cache directory to see if the converted model is already cached, has an appropriate modification date, etc).